runelite-client/src/main/java/net/runelite/client/plugins/party/PartyPanel.java

```diff
@@ -159,6 +159,7 @@ class PartyPanel extends PluginPanel
 					null,
 					null,
 					"");
+				s = s.toLowerCase()
```

Comment:
`generatePassphrase` uses `token.toLowerCase(Locale.US)` so probably the locale should be specified here, too.